### PR TITLE
[CU-868b7b8b9] Upload Process nits & Improvements.

### PR DESCRIPTION
**Pull request by lrojas94**

### CODE REVIEW

File: src/components/Image/ImageDropzone/MediaDropzone.tsx [hunk 1]
```diff
@@ -55,6 +56,11 @@ export function MediaDropzone({
         constants.mediaUpload.maxVideoFileSize
       : settings.maxSize
     : constants.mediaUpload.maxVideoFileSize;
+
+  const seconds = settings?.maxVideoDuration ?? constants.mediaUpload.maxVideoDurationSeconds;
+  const durationLabel =seconds > 60 ? dayjs
+    .duration(seconds, 'seconds')
+    .format(`mm [minutes (${seconds} seconds)]`) : `${seconds} seconds`;
```
Comment (manuelurenah) [inline src/components/Image/ImageDropzone/MediaDropzone.tsx:63]: I think we have a couple helper functions for this, try the `formatSeconds` from `number-helpers.ts`

File: src/server/controllers/post.controller.ts [hunk 2]
```diff
@@ -179,6 +180,26 @@ export const updatePostHandler = async ({
         throw throwBadRequestError('You must select a tag for this collection');
       }
 
+      if (collection.metadata.entriesRequireTitle && !post.title) {
+        throw throwBadRequestError('Your entry must have a title to be submitted');
+      }
+
+      if (collection.metadata.entriesRequireTools) {
+        // Check all images within this post has tools:
+        const exists = await dbWrite.image.findFirst({
```
Comment (manuelurenah) [inline src/server/controllers/post.controller.ts:189]: Should we use dbRead instead?
```suggestion
        const exists = await dbRead.image.findFirst({
```

File: src/server/services/collection.service.ts [hunk 3]
```diff
@@ -1431,6 +1431,46 @@ export const updateCollectionItemsStatus = async ({
   if (!manage && !isOwner)
     throw throwAuthorizationError('You do not have permissions to manage contributor item status.');
 
+  const collectionMetadata = collection.metadata as CollectionMetadataSchema;
+
+  if (status === CollectionItemStatus.ACCEPTED) {
+    if (collectionMetadata?.judgesCanScoreEntries) {
+      const exists = await dbWrite.collectionItem.findFirst({
```
Comment (manuelurenah) [inline src/server/services/collection.service.ts:1438]: Same, should it be dbRead? Just concerned with connection limit or overloading the write db

File: src/server/services/collection.service.ts [hunk 4]
```diff
@@ -1431,6 +1431,46 @@ export const updateCollectionItemsStatus = async ({
...
+
+      if (exists) {
+        throw throwBadRequestError(
+          'Some of the items selected do not have scores. Please ensure all items have scores before approving them.'
+        );
+      }
+    }
+
+    if (collectionMetadata?.judgesApplyBrowsingLevel && collection.type === CollectionType.Image) {
+      const exists = await dbWrite.collectionItem.findFirst({
```
Comment (manuelurenah) [inline src/server/services/collection.service.ts:1455]: Ditto

File: src/server/services/collection.service.ts [hunk 5]
```diff
@@ -1431,6 +1431,46 @@ export const updateCollectionItemsStatus = async ({
...
+            nsfwLevel: {
+              in: [0, -1],
+            },
+          },
+        },
+      });
+
+      if (exists) {
+        throw throwBadRequestError(
+          'Some of the items selected have not been given an NSFW rating. Please ensure all items have an NSFW rating before approving them.'
```
Comment (manuelurenah) [inline src/server/services/collection.service.ts:1468]:
```suggestion
          'Some of the items selected have not been given a NSFW rating. Please ensure all items have a NSFW rating before approving them.'
```

**Review by manuelurenah — APPROVED** [review]

File: src/components/Image/ImageDropzone/MediaDropzone.tsx (hunk 1 above)
Comment (lrojas94) [reply src/components/Image/ImageDropzone/MediaDropzone.tsx:63]: I know, but because I didn't want my task returned did it exactly as Matty wanted it 😬

**Pull request merged**